tests/integration/test_record_mode.py

```diff
@@ -9,16 +8,21 @@ def test_once_record_mode(tmpdir):
     with vcr.use_cassette(testfile, record_mode="once"):
         # cassette file doesn't exist, so create.
         response = urlopen('http://httpbin.org/').read()
```

Comment:
Yeah, if we're not doing anything with the variable I suppose we don't need to create it.  (BTW thanks a lot for doing this!  Having all the code be pep8 gives a nice clean feeling :) :+1: